### Варіант № 3
### Завдання № 379(г)
Дана дійсна матриця розміру $m\times n$. Визначити числа
$b_1,...,b_m$, які рівні відповідно значенням середніх
арифметичних елементів рядків.

In [2]:
from random import random

try:
    m = int(input("Enter the number of rows:"))
    n = int(input("Enter the number of columns:"))
    matrix = [[round(random() * 10, 1) for j in range(n)]for i in range(m)]
    for i in range(m): print(matrix[i])

    print("Average values of rows:\n",
          [round(sum(i) / n, 1) for i in matrix])
except ValueError:
    print("Input error!")

[8.2, 2.5, 7.6, 5.9, 2.9, 7.0, 2.5, 8.8, 8.0, 8.7]
[8.6, 4.4, 5.4, 6.1, 5.1, 1.5, 3.7, 3.9, 7.7, 3.5]
[6.1, 2.4, 7.6, 1.1, 1.3, 8.8, 0.6, 3.2, 1.1, 3.7]
[8.6, 4.3, 4.2, 0.4, 2.6, 9.7, 7.6, 3.1, 2.8, 8.8]
[9.2, 9.3, 6.6, 5.1, 0.8, 2.2, 9.8, 0.5, 2.3, 3.0]
[5.6, 3.4, 5.9, 5.4, 9.4, 8.2, 8.7, 7.6, 2.5, 4.8]
[5.0, 8.1, 5.8, 1.4, 2.1, 2.6, 5.6, 5.7, 4.3, 8.6]
[0.8, 2.4, 1.0, 8.5, 1.2, 3.9, 5.1, 2.8, 2.5, 2.3]
[5.1, 0.5, 0.2, 7.4, 1.0, 8.4, 8.7, 8.4, 4.4, 3.2]
[7.0, 5.3, 7.9, 5.9, 4.4, 5.2, 9.6, 4.9, 1.2, 3.8]
Average values of rows:
 [6.2, 5.0, 3.6, 5.2, 4.9, 6.2, 4.9, 3.0, 4.7, 5.5]


# Examples

### Example 1
[7.4, 0.5, 4.7, 3.8, 0.6]
[9.8, 7.1, 5.1, 6.3, 4.4]
[0.9, 6.4, 4.4, 5.7, 0.5]
[6.7, 4.8, 5.2, 5.6, 8.9]
[8.5, 0.5, 6.9, 2.7, 4.3]
Average values of rows:
 [3.4, 6.5, 3.6, 6.2, 4.6]
### Example 2
[0.4, 2.7, 0.3, 6.7, 0.7]
[2.3, 7.2, 2.6, 6.5, 4.8]
[0.1, 7.5, 5.5, 9.0, 6.6]
[4.9, 3.6, 2.9, 2.1, 0.0]
[3.4, 3.6, 9.7, 7.0, 5.8]
Average values of rows:
 [2.2, 4.7, 5.7, 2.7, 5.9]
### Example 3
[5.7, 2.8, 1.8, 5.2, 7.2]
[9.4, 5.9, 4.7, 9.2, 0.4]
[6.2, 5.4, 5.7, 3.9, 9.6]
[4.2, 8.5, 3.4, 9.0, 9.0]
[5.5, 2.3, 4.1, 5.4, 8.2]
Average values of rows:
 [4.5, 5.9, 6.2, 6.8, 5.1]
